.github/workflows/integration_test.yml

```diff
@@ -1,29 +1,24 @@
 name: Integration test
 
 on:
-  pull_request:
-    paths-ignore:
-      - 'doc/**'
-      - 'enhancements/**'
-      - '*.md'
+  workflow_call:
 
 env:
-  GO_VERSION: "1.18"
   OUTPUT_DIR: "_output/"
   FILE_NAME: "kepler.tar.gz"
   ARTIFACT_DIR: "/tmp/artifacts"
 
 jobs:
   build-kepler:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - name: checkout source
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: install Go
         uses: actions/setup-go@v3
         with:
-          go-version: ${{env.GO_VERSION}}
+          go-version-file: go.mod
```

Comment:
Since we have aligned the go version to go.mod, shall we bump the go version in go.mod to latest then?
i.e 1.18->1.21

Reply:
How about in a different PR so that the changes introduced by this PR is minimal and can be merged?

Reply:
+1 have a min PR

Reply:
Resolve as this should be in another PR.